package.json

```diff
@@ -1,32 +1,71 @@
+  "description": "A tool to help Polymer team manage repos.",
   "private": true,
   "scripts": {
-    "clean:client": "cd ./client/ && rm -rf ./node_modules && npm install --only=production",
-    "clean:server": "cd ./server/ && rm -rf ./node_modules && npm install --only=production && cd node_modules && echo num dep files before prune: $(find . -type f | wc -l) && find . -type f -not \\( -name '*.js' -or -name '*.json' -or -name '*.node' \\) -delete && echo num dep files after prune: $(find . -type f | wc -l)",
-    "clean": "npm run clean:client && npm run clean:server",
-    "build:client": "cd ./client/ && npm run build",
-    "build:server": "cd ./server/ && npm run build",
-    "build": "npm run build:client && npm run build:server",
-    "npm-install:client": "cd ./client/ && npm install",
-    "npm-install:server": "cd ./server/ && npm install",
-    "npm-install": "npm run npm-install:client && npm run npm-install:server",
-    "install": "npm run npm-install",
-    "deploy:gcloud": "gcloud app deploy app.yaml --project google.com:project-health-internal --quiet",
-    "deploy": "npm run build && npm run clean && npm run deploy:gcloud && npm run npm-install",
-    "format": "find server/src client/*.ts -name '*.ts' | xargs clang-format --style=file -i",
-    "precommit": "npm run format",
-    "server": "npm run build && node server.js"
+    "generate-gql-types": "apollo-codegen generate \"src/**/*.ts\" --schema \"src/types/github-schema.json\" --target typescript --addTypename --output \"src/types/gql-types.ts\"",
+    "clean": "rm -rf ./build/",
+    "copy": "cd src && cp --parents **/*.{html,json} ../build && cp --parents **/**/*.js ../build && cp -r --parents server/test/replays/**/* ../build",
+    "build:cli": "tsc -p src/cli/tsconfig.json --outDir build",
+    "build:client": "tsc -p src/client/tsconfig.json --outDir build",
+    "build:server": "tsc -p src/server/tsconfig.json --outDir build",
+    "build": "npm run clean && npm run generate-gql-types && npm run build:client && npm run build:cli && npm run build:server && npm run copy && npm run lint",
+    "build:watch": "npm run build && (npm run build:server -- --watch & npm run build:client -- --watch & npm run build:cli -- --watch)",
+    "lint": "tslint src/**/*.ts -t stylish",
+    "check-coverage": "nyc check-coverage --lines 70 --functions 70",
+    "report-coverage": "nyc report --reporter=text-lcov | coveralls",
+    "test": "npm run build && RECORD=false npm run test:ava -- --timeout=10sec && npm run check-coverage",
+    "test:record": "npm run build && RECORD=true npm run test:ava -- --timeout=1min",
+    "test:ava": "nyc --reporter=text --reporter=html ava build/*/test/**/*-test.js --verbose"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/PolymerLabs/project-health.git"
   },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@types/command-line-args": "^4.0.2",
```

Comment:
I think it's ok to put `@types` in `devDependencies` because this is not a library, as discussed at https://github.com/Microsoft/types-publisher/issues/81. (No action required)